openslides/assignments/static/js/assignments/site.js

```diff
@@ -356,13 +356,13 @@ angular.module('OpenSlidesApp.assignments.site', [
         $scope.sort.column = 'title';
         $scope.sortOptions = [
             {name: 'agenda_item.getItemNumberWithAncestors()',
-             display_name: 'Item'},
+             display_name: gettext('Item')},
             {name: 'title',
-             display_name: 'Title'},
-            {name: 'open_posts',
-             display_name: 'Open posts'},
+             display_name: gettext('Title')},
             {name: 'phase',
-             display_name: 'Phase'},
+             display_name: gettext('Phase')},
+            {name: 'assignment.assignment_related_users.length',
```

Comment:
@FinnStutzenstein I changed the sort field `open_posts` to number of candidates which is more useful. But with this name (`assignment.assignment_related_users.length`) the sorting by number is not correct. Do you know whats wrong?